components/event-system/queue.js

```diff
@@ -0,0 +1,91 @@
+const HOW_MANY_PRIORITIES_ARE_THERE_NITRO = 6
+const eventQueue = Array.from(Array(HOW_MANY_PRIORITIES_ARE_THERE_NITRO), () => [])
+const eventContentListeners = []
+let eventContentOnce = []
+
+function eventQueueEmpty () {
+  for (const queue of eventQueue) {
+    if (queue.length) {
+      return false
+    }
+  }
+  return true
+}
+
+function notifyOfContent () {
+  for (const callback of eventContentListeners) {
+    try {
+      callback()
+    } catch (error) {
+      error.message = 'NOOP'
```

Comment:
We should at least `console.error` these errors. They may be extremely unlikely, but without strict types, errors can come from *anywhere*. 👀